### PR TITLE
chore(BankIdV6Dialog): emotion --> vanilla-extract-css

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## Describe your changes

- Use `vanilla-extract-css` styles for `BankIdV6Dialog`

## Justify why they are needed

Take the opportunity of doing the migration for this new component.

### CODE REVIEW

**Comment by guilhermespopolin** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #3928** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3928?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3928?utm_source=stack-comment).

**Review by alebedev — APPROVED** [review]
Great!

File: apps/store/src/components/BankIdV6Dialog/BankIdV6Dialog.tsx [hunk 1]
```diff
@@ -52,12 +59,17 @@ export const BankIdV6Dialog = () => {
         }
 
         Content = (
-          <ContentWrapper maxWidth="35rem">
+          <div
+            className={contentWrapper}
+            style={assignInlineVars({
+              [contentWrapperMaxWidth]: '35rem',
```
Comment (alebedev) [inline apps/store/src/components/BankIdV6Dialog/BankIdV6Dialog.tsx:65]: Nice!

**Comment by guilhermespopolin** [comment]
### Merge activity

* **Feb 15, 7:40 AM EST**: @guilhermespopolin started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3928).

**Pull request merged**